apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -38,7 +38,9 @@
     channel_health_check/2,
     add_channel/3,
```

Comment:
Q: are the sync APIs still needed ?

Reply:
there are some tests calling it (most shouldn't :melting_face: ), but dry run helper also calls it, and it's appropriate for dry run, I think